experimental/PropertyDDS/examples/property-inspector/.eslintrc.js

```diff
@@ -4,16 +4,16 @@
  */
 
 module.exports = {
-	extends: ["@fluidframework/eslint-config-fluid"],
+    extends: ["@fluidframework/eslint-config-fluid/minimal"],
     "parserOptions": {
         "project": ["./tsconfig.json"]
     },
-	rules: {
-		"@typescript-eslint/strict-boolean-expressions": "off",
+    rules: {
+        "@typescript-eslint/strict-boolean-expressions": "off",
         "import/no-internal-modules": "off",
         "unicorn/filename-case": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
         "@typescript-eslint/unbound-method": "off",
         "import/no-unassigned-import": "off"
-	},
```

Comment:
Formatting is applied in this file. Is it okay to run formatting when we are bumping `eslint` to `2.0.0`?

Reply:
Oh, no, good catch. I'll revert the formatting changes here.

Reply:
Hmm, actually, it looks like this file had a mix of tabs and spaces. I think I might actually leave these changes since they at least make the space formatting consistent.